### PR TITLE
[plugin.video.ardaktuell] 1.0.1

### PR DESCRIPTION
**Pull request by Heckie75**
### Description
v1.0.1 (2021-06-02)
- Fixed broken URLs to github
- Fixed issue #2: Navigation back from episodes menu is incorrect
- Refactoring: seperate general rss code for easier maintainance in future
- Hint: restart is required in order to load moved translations

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            "name": "ARD Mal angenommen...",
+            "icon": "https://www.tagesschau.de/multimedia/bilder/mal-angenommen-podcast-cover-105~_v-original.jpg",
+            "audio_url": "https://www.tagesschau.de/multimedia/podcasts/mal-angenommen-feed-101.xml",
+            "type": "audio"
+        }
+    ]
+
+    def __init__(self, addon_handle):
+
+        AbstractRssAddon.__init__(self, self.__PLUGIN_ID__, addon_handle)
```
Comment (romanvm) [inline plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py:67]: `super().__init__(self.__PLUGIN_ID__, addon_handle)`. It's Python 3 after all :-)

File: plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+        def _get_node_by_path(path):
+
+            if path == "/":
+                return dir_structure[0]
+
+            tokens = path.split("/")[1:]
+            node = dir_structure[0]
+
+            while len(tokens) > 0:
```
Comment (romanvm) [inline plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py:119]: `while tokens:`

**Review by romanvm — APPROVED** [review]
LGTM. Code comments are not mandatory.

File: plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py (hunk 1 above)
Comment (Heckie75) [reply plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py:67]: I'm going to change it next time.

File: plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py (hunk 2 above)
Comment (Heckie75) [reply plugin.video.ardaktuell/resources/lib/ardaktuell/ardaktuelladdon.py:119]: Also, here. Will be changed next time.

**Comment by Heckie75** [comment]
Is there anything left to do from my side?

**Pull request merged**